internal/display/bufwindow.go

```diff
@@ -495,7 +520,7 @@ func (w *BufWindow) displayBuffer() {
 		bloc.X = bslice
 
 		// returns the rune to be drawn, style of it and if the bg should be preserved
-		getRuneStyle := func(r rune, style tcell.Style, isplaceholder bool) (rune, tcell.Style, bool) {
+		getRuneStyle := func(r rune, style tcell.Style, drawoffset int, linex int, isplaceholder bool) (rune, tcell.Style, bool) {
```

Comment:
Any reason why you renamed `showoffset` to `drawoffset`?
Neither name seems quite intuitive to me, but at least `showoffset` somehow relates to `showchars` (and `showchars` is the only reason for the existence of this argument, right?).